include/spdk/nvme.h

```diff
@@ -387,9 +387,10 @@ struct spdk_nvme_qpair;
  * detected on a request.
  */
 typedef void (*spdk_nvme_timeout_cb)(struct spdk_nvme_ctrlr *ctrlr,
-				     struct spdk_nvme_qpair *qpair,
-				     void *cb_arg);
-
+				     void *cb_arg,
+				     uint16_t abort_count,
+				     uint16_t cid,
+				     uint16_t sqid);
```

Comment:
I think passing back a spdk_nvme_qpair pointer rather than sqid would be best, and then we can make the abort command function take a qpair pointer as well.